components/classes/StudentListTableBody.tsx

```diff
@@ -78,6 +78,17 @@ const StudentListTableBody: StylableFC<{
             </td>
           )}
 
+          {/* Enrollment via randomization */}
+          {options.columns.includes("randomized") && (
+            <td className="!border-l-0">
+              {student.chosen_elective?.randomized_students.some(
+                (randomizedStudent) => student.id === randomizedStudent.id,
+              )
+                ? "*"
```

Comment:
Putting just a "*" as a mark that the student is randomly enrolled is good and subtle, but not too subtle for the user!

The if statement here for " " or "*" is put to good use and simple too!